.env

```diff
@@ -0,0 +1,10 @@
+YELP_CONSUMER_KEY: RC-dbM2zuKUQmpuXSeYp5A
+YELP_CONSUMER_SECRET: O_YOLmwe2o1IwVsl4-hUVvQn0dM
+YELP_TOKEN: KERJPZzNUilFm-RRMdk9V-90kxzYO213
+YELP_TOKEN_SECRET: uLfJRgRSyUporozYTKebsZ81Asc
+
+
+
+CLIENT_ID: 84d3407e199f4c9bbbfe6e6e51b9fe0a
+
+CLIENT_SECRET: 84c0798ce9b94370a384d263efccc892
```

Comment:
Oops. Don't want these exposed on github! I think you tried to gitignore them, but it didn't quite happen.